Basic-Car-Maintenance/Shared/Dashboard/ViewModels/DashboardViewModel.swift

```diff
@@ -79,6 +75,10 @@ class DashboardViewModel {
         }
     }
     
+    func syncEvents() async {
+        await self.getMaintenanceEvents()
+    }
+    
```

Comment:
I don't think it's necessary to create a new function that just calls another function in the same scope